diff --git a/flexible_resnet.py b/flexible_resnet.py
index 6eaa010..7539293 100644
--- a/flexible_resnet.py
+++ b/flexible_resnet.py
@@ -426,9 +426,9 @@ def show_training_time(start, end):
 
 def main():
     # Number of epochs
-    num_epochs = 45
+    num_epochs = 50
     # Dataset
-    dataset = DataSet.CIFAR10
+    dataset = DataSet.STL10
     # Number of layers for the resnet model
     num_layers = 18
     # Custom ResNet model
@@ -451,7 +451,7 @@ def main():
     # 60% acc on stl10
     lr_milestones = [(0, 0.01), (10, 0.1), (20, 0.18), (30, 0.1), (40, 0.01), (50, 0)]
 
-    #lr_milestones = [(0, 0.01), (10, 0.1), (20, 0.18), (30, 0.1), (40, 0.01), (50, 0)]
+    lr_milestones = [(0, 0.01), (10, 0.1), (20, 0.18), (30, 0.1), (40, 0.01), (50, 0)]
 
     #lr_milestones = [(0, 0.1), (num_layers/2, 0.2), (20, 0.1), (30, 0.01), (50, 0)]
     adjust_lr = True